packages/account/src/providers/transaction-summary/input.ts

```diff
@@ -33,16 +34,23 @@ export function getInputsContract(inputs: Input[]) {
 }
 
 /** @hidden */
-export function getInputFromAssetId(inputs: Input[], assetId: string) {
+export function getRelevantInputs(inputs: Input[], assetId: string = ZeroBytes32) {
```

Comment:
the previous name was better imo:
1- it was more explicit, this method exists to return the input funding a specific asset to the transaction
2- getRelevantInputs is too generic. "relevant" to what?

Reply:
7004319a6